### PR TITLE
Fix command for installing lts version

### PR DESCRIPTION
**Pull request by mic-max**
nvm install --lts
Unrecognized version: "--lts"

### CODE REVIEW

**Comment by chalin** [comment]
Would you mind copy-pasting the exact command invocation, the output and it's version?

According to the `nvm` command help, `--lts` _is_ a valid flag:

```console
$ nvm version
v16.13.0
$ nvm | grep 'latest LTS version'
  nvm install --lts                     Install the latest LTS version
  nvm use --lts                         Use the latest LTS version
```

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,7 +7,11 @@ Follow these steps for local or [cloud-IDE][] (via [Gitpod.io][]) development:
 - Fork and clone this repository (for local development only).
 - Install the latest [LTS release][] of **Node**, using **[nvm][]** for example:
   ```console
-  $ nvm install --lts
```
Comment (chalin) [inline CONTRIBUTING.md:10]: The original command invocation is correct according to the `nvm` help info and docs.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,7 +7,11 @@ Follow these steps for local or [cloud-IDE][] (via [Gitpod.io][]) development:
...
+  $ nvm install lts
+  ```
+- Use the latest LTS release
+  ```console
+  $ nvm use lts
```
Comment (chalin) [inline CONTRIBUTING.md:14]: You don't need to run `use` after ` install` as far as I recall.

**Review by chalin — CHANGES_REQUESTED** [review]

**Comment by chalin** [comment]
Hi @mic-max. I'm going to close this for now. If you still feel that there's something wrong with the command invocation, feel free to reopen the PR.

**Comment by mic-max** [comment]
Hey @chalin so I was running `nvm` on Windows, but I guess the instructions here are all for Linux, that explains the difference

**Comment by chalin** [comment]
Thanks for the feedback @mic-max. It's unfortunate that the `nvm` command for Windows accepts different arguments. I'll see how we can capture that in the instructions.

**Comment by chalin** [comment]
I'll follow up via #1071. Thanks again for letting us know about this.

**Pull request closed without merging**